### PR TITLE
fix port forwarding

### PR DESCRIPTION
**Pull request by jakeschuurmans**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-JAEGER_DASH_PORT_FW=16686:16686
-OTEL_PORT_FW=4317:4317
-MINIO_PORT_FW=9000:9000
+
+WITH_LAN=
+CONDITION_API_PORT_FW=9001
+CONDITION_ORC_API_PORT_FW=9001
+ALLOY_PORT_FW=9091
+FLEETDB_PORT_FW=8000
+CRDB_UI_PORT_FW=8080
```
Comment (coffeefreak101) [inline Makefile:10]: Maybe we should keep CRDB UI as `8081:8080` since 8080 conflicts with FLEETDB.

Comment (coffeefreak101) [reply Makefile:10]: Looks like CRDB does use 8080:
https://www.cockroachlabs.com/docs/stable/ui-overview#db-console-access
And this was 8081 and then fixed to 8081:8080:
https://github.com/metal-toolbox/sandbox/commit/c1690d2016f27e8672a4f0c5af5d9e3092655ddd
So changing it to 8080 should be fine.

File: Makefile [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
-CRDB_UI_PORT_FW=8081:8080
-CRDB_PORT_FW=26257:26257
-CHAOS_DASH_PORT_FW=2333:2333
-JAEGER_DASH_PORT_FW=16686:16686
-OTEL_PORT_FW=4317:4317
-MINIO_PORT_FW=9000:9000
+
+WITH_LAN=
+CONDITION_API_PORT_FW=9001
+CONDITION_ORC_API_PORT_FW=9001
```
Comment (coffeefreak101) [inline Makefile:7]: These two conflict though, so maybe we should keep condition orc `9002:9001`.

Comment (jakeschuurmans) [reply Makefile:7]: i think you have proved that my "simple fix" for some things here is not viable. Im going to reject this PR and try again.

**Pull request closed without merging**